sql/common/src/main/scala/org/apache/sedona/sql/UDF/Catalog.scala

```diff
@@ -102,7 +102,7 @@ object Catalog {
     function[ST_ClosestPoint](),
     function[ST_Boundary](),
     function[ST_MinimumBoundingRadius](),
-    function[ST_MinimumBoundingCircle](BufferParameters.DEFAULT_QUADRANT_SEGMENTS),
```

Comment:
Please update the doc in Sedona Spark saying that the default param has been changed.